tutorial-outline.md

```diff
@@ -0,0 +1,215 @@
+# Olmsted Screen Capture Tutorial Outline
+
+## Part 1: Olmsted Web Application Features (15-20 minutes)
+
+### 1.1 Accessing Olmsted (2-3 minutes)
+**Demo all three access methods:**
+- **Option A**: Public website at https://olmstedviz.org
```

Comment:
Is there a reason not to use the public website? All the data is client-side irrespective...